src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/design/finalclass/InputFinalClassClassWithPrivateCtorWithNestedExtendingClass.java

```diff
@@ -10,17 +10,18 @@
 public class InputFinalClassClassWithPrivateCtorWithNestedExtendingClass {
     class A {
         private A() {}
-        private class ExtendA extends A {}
+        private final class ExtendA extends A {}
```

Comment:
I don't think we should be changing existing inputs. I think @romani will agree.

Reply:
Yes , we don't do updates in existing Inputs, they are wrong or weird by design.

Reply:
done